ispyb/model/screening.py

```diff
@@ -0,0 +1,394 @@
+from __future__ import absolute_import, division, print_function
+
+import ispyb.model
+from ispyb.model.integration import UnitCell
+
+
+class Screening(ispyb.model.DBCache):
+    """An object representing a Screening database entry.
+    The object lazily accesses the underlying database when necessary and
+    exposes record data as python attributes.
+    """
+
+    def __init__(self, screening_id, db_conn, preload=None):
+        """Create a Screening object for a defined ScreeningId.
+
+        Requires a database connection object exposing further data access
+        methods.
+
+        :param sceening_id: ScreeningId
+        :param db_conn: ISPyB database connection object
+        :return: A Screening object representing the database entry for
+                 the specified ScreeningId
+        """
+        self._db = db_conn
+        self._screening_id = int(screening_id)
+        if preload:
+            self._data = preload
+
+    def reload(self):
+        """Load/update information from the database."""
+        raise NotImplementedError()
+
+    @property
+    def screening_outputs(self):
```

Comment:
given that you have a `screenings` type object we can probably just call this property `.outputs()`